### PR TITLE
Update popup code

### LINKED ISSUE #216: Better Standalone Example For Creating a Popup
Examples can be frustrating if they are not fully self contained.  The create pop-up examples are ones that one cannot be just copy and pasted into the editor and run.

So this example, unlike the first `Button` code example, does not need a code-behind `Click` event to show the popup.  Instead the below example uses a `ToggleButton` where the popup's `IsOpen` state is bound to the toggle's property `IsChecked`; hence making it selfcontained:

```
<ToggleButton   x:Name="OpenState"
                 Height="30" Width="60">
    <StackPanel>
        <TextBlock Text="{Binding IsChecked, ElementName=OpenState}"
                    FontSize="18"
                    VerticalAlignment="Center"
                    HorizontalAlignment="Center">
        </TextBlock>

        <Popup Name="myPopup"
                IsOpen="{Binding IsChecked, ElementName=OpenState}">
            <TextBlock Name="myPopupText"
                        Background="LightBlue"
                        Foreground="Blue">Popup Text</TextBlock>
        </Popup>

    </StackPanel>
</ToggleButton>
```
Hence a fully contained example. :-)

---
Its frustrating to a new learner when they can't just run an example. As mentioned, the first example shown (https://docs.microsoft.com/en-us/dotnet/desktop/wpf/controls/popup-overview?view=netframeworkdesktop-4.8#creating-a-popup) does not work. My example above does work without code behind.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e046afa9-e4b1-1f27-88c3-be8540d6f64e
* Version Independent ID: 58b0db73-1b13-e2a5-cff3-fb89493422d1
* Content: [Popup Overview - WPF .NET Framework](https://docs.microsoft.com/en-us/dotnet/desktop/wpf/controls/popup-overview?view=netframeworkdesktop-4.8)
* Content Source: [dotnet-desktop-guide/framework/wpf/controls/popup-overview.md](https://github.com/dotnet/docs-desktop/blob/master/dotnet-desktop-guide/framework/wpf/controls/popup-overview.md)
* Product: **dotnet-framework**
* Technology: **dotnet-wpf**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

### PR DESCRIPTION
**Pull request by adegeo**
## Summary

- Recoded XAML for popup so that it doesn't need a event handler.
- Removed 2nd popup example that required more code behind to work. Not referenced nor useful right now.

Fixes #216 

@luisquintanilla @CheetahChrome

Preview Link

File Type | File Name | Published Url
-- | -- | --
Content | dotnet-desktop-guide/framework/wpf/controls/popup-overview.md | https://review.docs.microsoft.com/en-us/dotnet/desktop/wpf/controls/popup-overview?view=netframeworkdesktop-4.8&branch=pr-en-us-1111

### CODE REVIEW

**Review by CheetahChrome — APPROVED** [review]

**Comment by adegeo** [comment]
@luisquintanilla this one too :)

File: dotnet-desktop-guide/framework/wpf/controls/snippets/popup-overview/csharp/Window1.xaml.cs [hunk 1]
```diff
@@ -36,20 +36,20 @@ private void setColors(object sender, RoutedEventArgs e)
         }
         //</SnippetIsFocused>
 
-        //<SnippetCreatePopupCode>
+        //<CreatePopup>
```
Comment (luisquintanilla) [inline dotnet-desktop-guide/framework/wpf/controls/snippets/popup-overview/csharp/Window1.xaml.cs:39]: Is this snippet used anywhere? I don't see it any references to it so just want to make sure I didn't miss anything. Other than that, looks good to me.

**Review by luisquintanilla — APPROVED** [review]
@adegeo Looks good to me. Just a minor question.

File: dotnet-desktop-guide/framework/wpf/controls/snippets/popup-overview/csharp/Window1.xaml.cs (hunk 1 above)
Comment (adegeo) [reply dotnet-desktop-guide/framework/wpf/controls/snippets/popup-overview/csharp/Window1.xaml.cs:39]: No, just old copy\paste

**Pull request merged**